package/MDAnalysis/coordinates/core.py

```diff
@@ -67,8 +64,8 @@ def get_reader_for(filename, permissive=False, format=None):
     if format is None:
         format = util.guess_format(filename)
     format = format.upper()
-    if permissive and format == 'PDB':
-        return _READERS['Permissive_PDB']
+    if format == 'PDB':
```

Comment:
This whole statement should be removed. (Lines 67 & 68) because we'll just run through the try/except below.